#goal definition
float64 startPos
float64 endPos
uint8 traversals
bool nullCmd
uint8 imagePub
bool useDist
string mapName
---
#result definition
bool success
---
#feedback
